controllers/sriov-controller/Makefile

```diff
@@ -0,0 +1,41 @@
+REGISTRY_NAME = 192.168.80.240:4000/ligato
+IMAGE_VERSION = latest
+
+.PHONY: all sriov-controller nsm-generate-sriov-configmap mac-controller mac-sriov-configmap container push clean test
+
+ifdef V
```

Comment:
Remove this block before committing.

Reply:
Same answer as above, it will all go away.